web/src/api/helpers.ts

```diff
@@ -68,6 +68,18 @@ export function getBasePath() {
     : 'https://app-backend.electricitymap.org';
 }
 
+export function cacheBuster(): string {
+  const current_date = new Date();
+  const date_to_hour = new Date(
+    current_date.getFullYear(),
+    current_date.getMonth(),
+    current_date.getDate(),
+    current_date.getHours(),
+    current_date.getMinutes()
+  );
+  return date_to_hour.toISOString();
+}
```

Comment:
Wouldn't it be simpler to do something like this instead?

```suggestion
export function cacheBuster(): string {
  const currentDate = new Date();
  currentDate.setSeconds(0);
  currentDate.setMilliseconds(0);
  
  return currentDate.toISOString();
}
```

Reply:
It would!